modules/openapi-generator/src/main/resources/python/api_response.mustache

```diff
@@ -14,7 +14,7 @@ class ApiResponse(BaseModel, Generic[T]):
     status_code: StrictInt = Field(description="HTTP status code")
     headers: Optional[Dict[StrictStr, StrictStr]] = Field(None, description="HTTP headers")
     data: T = Field(description="Deserialized data given the data type")
-    raw_data: StrictBytes = Field(description="Raw data (HTTP response body)")
+    raw_data: Optional[StrictBytes] = Field(description="Raw data (HTTP response body)")
```

Comment:
This is not in fact allowed to be `None` I believe. It's more correct to assert that `response_data.data` is not `None` in `response_deserialize()`.